[Target File]: gradle/libs.versions.toml

[Diff]:
diff --git a/gradle/libs.versions.toml b/gradle/libs.versions.toml
index 99a018f774..49daf59f95 100644
--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -26,7 +26,7 @@ androidx-navigation = "2.8.5"
 androidx-navigation3-core = "1.0.0"
 androidx-window = "1.3.0"
 glide = "4.15.1"
-libsignal-client = "0.87.4"
+libsignal-client = "0.88.0"
 mp4parser = "1.9.39"
 accompanist = "0.28.0"
 nanohttpd = "2.3.1"
@@ -170,7 +170,7 @@ libsignal-client = { module = "org.signal:libsignal-client", version.ref = "libs
 libsignal-android = { module = "org.signal:libsignal-android", version.ref = "libsignal-client" }
 protobuf-gradle-plugin = { module = "com.google.protobuf:protobuf-gradle-plugin", version.ref = "protobuf-gradle-plugin" }
 signal-aesgcmprovider = "org.signal:aesgcmprovider:0.0.4"
-signal-ringrtc = "org.signal:ringrtc-android:2.65.0"
+signal-ringrtc = "org.signal:ringrtc-android:2.65.3"
 signal-android-database-sqlcipher = "org.signal:sqlcipher-android:4.6.0-S1"
 
 # Third Party